examples/auth/app/pages/index.test.tsx

```diff
@@ -1,27 +1,25 @@
-import {render} from "test/utils"
+import { render } from "test/utils"
+
 import Home from "./index"
+import { useCurrentUser } from "app/core/hooks/useCurrentUser"
 
-jest.mock("@blitzjs/core", () => ({
-  ...jest.requireActual("@blitzjs/core")!,
-  useQuery: () => [
-    {
-      id: 1,
-      name: "User",
-      email: "user@email.com",
-      role: "user",
-    },
-  ],
-}))
+jest.mock("app/core/hooks/useCurrentUser")
+const mockUseCurrentUser = useCurrentUser as jest.MockedFunction<typeof useCurrentUser>
 
 test("renders blitz documentation link", () => {
   // This is an example of how to ensure a specific item is in the document
   // But it's disabled by default (by test.skip) so the test doesn't fail
   // when you remove the the default content from the page
 
   // This is an example on how to mock api hooks when testing
+  mockUseCurrentUser.mockReturnValue({
+    id: 1,
+    name: "User",
+    email: "user@email.com",
+    role: "user",
+  })
 
-  const {getByText} = render(<Home />)
-  const element = getByText(/powered by blitz/i)
-  // @ts-ignore
-  expect(element).toBeInTheDocument()
+  const { getByText } = render(<Home />)
+  const linkElement = getByText(/Documentation/i)
+  expect(linkElement).toBeInTheDocument()
 })
```

Comment:
So some change in this file is still causing tests to fail.

Not sure if `jest.mock` needs to be before the import?

Reply:
Ah I see. Should the test syncing be a separate PR then? I can commit just the single change to satisfy typescript and we can update the auth tests later? 
Also I'm actually not sure how to trigger the CI/CD tests, is there a way to retrigger them and check if the changes I made are OK in the monorepo?

Reply:
it's up to you. If we get tests passing, then fine to keep the change here.

CI auto runs on each `git push`, but you can run these failing tests locally by running `yarn jest` inside `examples/auth`

Reply:
Ah, I see. I reverted back to the original tests, with the only change being the <object> before the tests. I tried to troubleshoot the index.test.tsx file but I'm actually ran into issues with the forgotpassword mutation as well, so I reverted that back to its original version too. I made sure to run yarn jest this time and all tests passed.

Reply:
Ok, that was on me, I forgot to push the changes I'd made after resolving a merge conflict, I reran the yarn jest command and eslint --fix and lint, and those passed and waiting for the results of the CI tests